src/tools/rbd_mirror/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ add_library(rbd_mirror_types STATIC
 set(rbd_mirror_internal
   ClusterWatcher.cc
   ImageDeleter.cc
+  ImageMapper.cc
```

Comment:
That's true, I guess -- fine to leave it. I should probably rebase #12364 and see if I can work around the recreated image race condition.